enclave-runner/src/tcs.rs

```diff
@@ -50,7 +20,7 @@ pub enum CoResult<Y, R> {
 
 #[derive(Debug)]
 pub struct Usercall<T: Tcs> {
-    tcs: T,
+    pub tcs: T,
```

Comment:
Undo this change. The enclave state of a TCS in the middle of the usercall is such that it must only be resumed, you can't use the TCS except for returning from the usercall. The private encapsulation by `Usercall` guarantees this. See individual notes at usage sites on how to handle this.